wandb/apis/public/runs.py

```diff
@@ -1470,3 +1472,47 @@ def beta_scan_history(
             use_cache=use_cache,
         )
         return beta_history_scan
+
+    def download_history_exports(
+        self,
+        download_dir: pathlib.Path | str,
+    ) -> tuple[list[pathlib.Path], bool]:
+        """Download any parquet history files for the run to the provided directory.
+
+        If the run contains live data (data that is not yet exported to parquet),
+        then the second return value will be True.
+
+        Args:
+            download_dir: The directory to download the history files to.
+
+        Returns:
+            A tuple containing the list of file names
+            and a boolean indicating if the run contains live data.
```

Comment:
Returning a frozen dataclass will be more maintainable and could be easier to use:

```python
download_result = run.download_history_exports(dir)

download_result.paths          # list[pathlib.Path]
download_result.has_live_data  # bool
```

This will allow us to return more information in the future if necessary.